### DISCUSSION TITLE
How to setup & direct subdomain to Squarespace site

### DISCUSSION THREAD

**Discussion by liz-o4d** [vercel/community#1794] [Help] [answered]
### Summary

I inherited a website with the domain at GoDaddy, deployment and DNS managed by Vercel (I think? Vercel and GoDaddy both say the nameservers are managed by the other), and content via Contentful. I am trying to set up a subdomain and point it (but not the primary domain) to a separate Squarespace website for an event. I've got the CNAME records from Squarespace that need to be added, but I'm not sure how to properly add the subdomain in GoDaddy and/or Vercel. GoDaddy won't let me add DNS records it since it says the nameservers are managed by Vercel. And Vercel gives the error "Error:The specified target is not a fully qualified domain name." when I try to enter the subdomain info from Squarespace. I feel like I must be missing a step to set up the subdomain. Any help or pointers would be greatly appreciated!

**Comment by amyegan** [comment, marked as answer]
Hi @liz-o4d! The first thing you need to do is figure out if you're using Vercel's or GoDaddy's nameservers. From your description, it sounds like you are using Vercel nameservers. That means the DNS record for your subdomain can be set from your Vercel Dashboard. Then, you'll need to make sure you're directing the subdomain to the right Squarespace subdomain with a valid CNAME record. 

We have a guide covering common subdomain record scenarios: [How do I point a subdomain to a service outside of Vercel?](https://vercel.com/guides/pointing-subdomains-to-external-services)

And Squarespace has some other resources that might help you:
- [Connecting a domain to your Squarespace site](https://support.squarespace.com/hc/en-us/articles/205812378-Connecting-a-domain-to-your-Squarespace-site)
- [Troubleshooting domain connections](https://support.squarespace.com/hc/en-us/articles/206541757-Troubleshooting-domain-connections)
- [DNS records for connecting third-party domains](https://support.squarespace.com/hc/en-us/articles/360035485391-DNS-records-for-connecting-third-party-domains)
- [Subdomains - Creating a website using a subdomain of an existing SquareSpace site](https://forum.squarespace.com/topic/193505-subdomains-creating-a-website-using-a-subdomain-of-an-existing-squarespace-site/)

**Reply by liz-o4d** [reply]
Thank you, this was matter was resolved with Squarespace's support.
